### PR TITLE
INDY-905: add tests for rotation of BLS keys by Trustee

### PR DESCRIPTION
**Pull request by ashcherbakov**
Signed-off-by: ashcherbakov <alexander.sherbakov@dsr-company.com>

### CODE REVIEW

File: indy_client/test/cli/test_node.py [hunk 1]
```diff
@@ -25,6 +25,32 @@ def test_add_same_node_without_any_change(be, do, newStewardCli,
...
+    doSendNodeCmd(do, newNodeVals,
+                  expMsgs=["node already has the same data as requested"])
+    exitFromCli(do)
+
+
+def test_add_same_node_with_changed_bls_by_trustee(be, do, trusteeCli,
+                                                   newNodeVals, newNodeAdded,
+                                                   nodeValsEmptyData):
+    '''
+    Checks that it's possible to update BLS keys (just alias and new key are required)
```
Comment (mzk-vct) [inline indy_client/test/cli/test_node.py]: Please update docstring to make it describe what happens it test

File: indy_client/test/cli/test_node.py [hunk 2]
```diff
@@ -25,6 +25,32 @@ def test_add_same_node_without_any_change(be, do, newStewardCli,
     exitFromCli(do)
 
 
+def test_add_same_node_without_any_change_by_trustee(be, do, trusteeCli,
```
Comment (mzk-vct) [inline indy_client/test/cli/test_node.py]: Is there a test for successful editing?

File: indy_common/test/auth/test_auth_error.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from plenum.common.constants import NODE, TRUSTEE, BLS_KEY, STEWARD
+
+from indy_common.auth import Authoriser
+
+
+def test_node_not_allowed_role_error():
```
Comment (mzk-vct) [inline indy_common/test/auth/test_auth_error.py]: Is there test for successful auth?

Comment (ashcherbakov) [reply indy_common/test/auth/test_auth_error.py]: Yes, recently added tests in `indy_common/test/auth` package (test_auth_node, test_auth_nym, etc.).

File: indy_client/test/cli/test_node.py (hunk 2 above)
Comment (ashcherbakov) [reply indy_client/test/cli/test_node.py]: Yes, recently added tests in `indy_common/test/auth` package (test_auth_node, test_auth_nym, etc.).

**Pull request merged**